lib/charms/opensearch/v0/opensearch_config.py

```diff
@@ -134,7 +135,7 @@ def set_node(
         if len(cm_ips) > 0:
             self._opensearch.config.put(self.CONFIG_YML, "discovery.seed_hosts", cm_ips)
 
-        if "cluster_manager" in roles and len(cm_ips) < 2:  # cluster NOT bootstrapped yet
+        if "cluster_manager" in roles and contribute_to_bootstrap:  # cluster NOT bootstrapped yet
```

Comment:
I'd rather being explicit here `contribute_to_bootstrap is True`. I have seen subtle bugs as python can promote unexpected behaviour if the variable is None, empty list, int , etc. I understand that we use typing, but being explicit can also improve readability

Reply:
I think we're pretty safe here, even if a variable is None, empty etc.. a check on that variable will evaluate as `False`.

Reply:
IMO one of the nicest things about Python is that you can use these syntactic-sugar shortcuts.
Using `is` is already a shortcut, whereas being explicit would be `contribute_to_bootstrap == True`

As typing is enforced by our linting tooling, those subtle bugs are much less likely. Charm devs use static type checkers pretty ubiquitously (e.g Pylance in VSCode or similar), I think this is OK.

Example peek:

![image](https://user-images.githubusercontent.com/51744472/206546128-207cd48b-9725-446c-b1a0-e73bf40b9e57.png)

Reply:
@marcoppenheimer Just a small comment that sometimes can make a difference: "is" is not the same as "==". The two  mean different thing: "is" checks whether two variables point to the same object, "==" checks whether two objects are equal (as specified by the __eq__ magic method). When using bools that works fine (since True, like 1, etc, generally native types, support singletons), but this is generally not true